### ISSUE TITLE
postprocess jupyter html to add mathjax changes

### ISSUE THREAD

**Issue by phaustin** [eoas-ubc/paged_trio#5]
make a new conda environment with the github master branch nbconvert as an editable install:

```
conda create -n cdp2 --clone cdp
conda activate cdp2
git clone https://github.com/jupyter/nbconvert.git
cd nbconvert
pip install -e .
```
Then modify the base template to include the mathjax changes and use it in a new nbconvert convert:

`jupyter nbconvert --to cdp-html test.ipynb`

or similar

- [ ] modify j2 templates to produce day5_quiz_students.html from [day5_quiz_students.ipynb](https://github.com/eoas-ubc/paged_trio/blob/d8d55d994dadb6a46951fb3dd81757694f85e6d4/testhtml/day5_quiz_students.ipynb)
- [ ] move the modified templates into our own writer called cdp-html

**Comment by phaustin** [comment]
See the discussion about the template redesign from version 5.7 here:

https://github.com/jupyter/nbconvert/pull/1056

**Comment by phaustin** [comment]
here's the mathjax jinja2 template you'll need to modify:
[mathjax.html.j2](https://github.com/jupyter/nbconvert/blob/f4c7daefe0e24055a92f185d6fd803abf909db2f/share/jupyter/nbconvert/templates/base/mathjax.html.j2)

**Comment by phaustin** [comment]
Added some nbconvert notes to the [Readme.md](https://github.com/eoas-ubc/paged_trio#working-with-nbconvert)

Added checklist items to this issue

**Comment by phaustin** [comment]
closed by 121c667
